packages/react-components/react-nav-preview/src/components/NavCategoryItem/NavCategoryItem.types.ts

```diff
@@ -22,23 +22,23 @@ export type NavGroupSlots = {
 /**
  * Any value that identifies a specific navGroup.
```

Comment:
```suggestion
 * Any value that identifies a specific NavCategoryItem.
```

Reply:
Similar fixes needed elsewhere in this file as well.

Reply:
Yep - looks like my 'match whole word' toggle was selected. Should be updated now.